src/SecureElementReader.App/Views/Dialogs/VerifyPinDialog.axaml

```diff
@@ -1,48 +1,83 @@
+			<Setter Property="FontWeight" Value="Bold"/>
+			<Setter Property="Foreground" Value="Green"/>
+		</Style>
+
+		<Style Selector="Label.Alert">
+			<Setter Property="FontSize" Value="16"/>
+			<Setter Property="FontWeight" Value="Bold"/>
+			<Setter Property="Foreground" Value="Red"/>
+		</Style>
+
+	</Window.Styles>
+	
+	
+	
+	
   
-	<Grid>
-
-		<StackPanel Margin="10,0">
-		  <Label Margin="0,4,0,0" Content="{x:Static p:Resources.VerifyPin}"/>
-      <Rectangle HorizontalAlignment="Stretch" Fill="Olive" Height="2" Margin="0,4"/> 
-      <StackPanel Orientation="Horizontal" Spacing="10" Margin="0,5">      
-        <TextBox Watermark="{x:Static p:Resources.EnterPIN}" 
-                 Width="150"
-                 VerticalAlignment="Top" 
-                 Text="{Binding Pin}"
-                 PasswordChar="•" 
-                 Margin="0,-3" CornerRadius="0"/>
-	      <Button VerticalAlignment="Top" 
-                Command="{Binding VerifyCommand}" 
-                VerticalContentAlignment="Center" 
-                MinHeight="32" Margin="0,-3" CornerRadius="0" BorderThickness="0" 
-                Background="#19aa8c" Foreground="White" 
-                Content="{x:Static p:Resources.Verify}"/>      
-      </StackPanel>	  
-      <Rectangle HorizontalAlignment="Stretch" Fill="Gray" Height="2" Margin="0,4" />
-      <Label Content="{Binding SeResult}" Foreground="{Binding SeResultColor}" FontSize="16" FontWeight="Bold"/>
-      <Label Content="{Binding PkiResult}" Foreground="{Binding PkiResultColor}" FontSize="16" FontWeight="Bold"/>
-      <Label Content="{Binding TaxMessage}" IsVisible="{Binding ShowTaxMessage}" Foreground="Red" FontSize="16" FontWeight="Bold"/>
-    </StackPanel>
+	<Grid x:Name="BaseVerifyPinWindow" Width="500" Height="200">
+		<Grid x:Name="DefinedGrid" ColumnDefinitions="100,100,100,100,100" RowDefinitions="auto,auto,auto,auto">
+
+			<Label Grid.Row="0" Grid.ColumnSpan="5" Height="25" Margin="5" HorizontalContentAlignment="Left" Content="{DynamicResource VerifyPin}"/>
+
+			<StackPanel  Grid.Row="1" Grid.ColumnSpan="5" Orientation="Horizontal" Height="25" Spacing="10" Margin="10,5">
+				<TextBox Watermark="{DynamicResource EnterPIN}"
+						 Width="150"
+						 VerticalAlignment="Top"
+						 Text="{Binding Pin}"
+						 PasswordChar="•"
+						 Margin="0,-3" CornerRadius="0"/>
+				<Button VerticalAlignment="Top"
+						Command="{Binding VerifyCommand}"
+						VerticalContentAlignment="Center"
+						MinHeight="32" Margin="0,-3" CornerRadius="0" BorderThickness="0"
+						Background="#19aa8c" Foreground="White"
+						Content="{DynamicResource Verify}"/>
+			</StackPanel>
+
+			<Rectangle Grid.Row="2" Grid.ColumnSpan="5" HorizontalAlignment="Stretch" Fill="Gray" Height="2" Margin="10,5,10,0" />
+
+			<StackPanel x:Name="Labels" Grid.Row="3" Grid.ColumnSpan="5" Height="120" Margin="10,5,10,0">
+				<Label Content="{DynamicResource PleaseInsertPIN}" IsVisible="{Binding PleaseInsertPIN}" Classes="Alert"/>
```

Comment:
Visible properties not set right 

![image](https://user-images.githubusercontent.com/68274836/199218583-f54123f4-c223-4683-9778-271e27ff2c7d.png)